src/components/Donate.jsx

```diff
@@ -0,0 +1,59 @@
+// MyStoreCheckout.js
+import React, { Component } from 'react';
+import AsciiOdie from './AsciiOdie'
+import Loading from './Loading'
+import DonateForm from './DonateForm'
+import {StripeProvider} from 'react-stripe-elements';
+import {Elements} from 'react-stripe-elements';
+import {StripeKeys} from '../utilities/stripeKeys';
+
+class Donate extends Component<{}, {stripe: null | StripeShape}> {
```

Comment:
What does `{}, {stripe: null | StripeShape}> {` do?

Reply:
No clue. that is from their example code for injecting stripe.js

Reply:
so cryptic